Feature/modeling and vis and tracking (#1)
snowplow/consent-accelerator: content/tracking/_index.md

+++
title = "Tracking"
chapter = true
weight = 2
pre = "1. "
post = ""
+++

<!-- ### Chapter 4 -->

# Tracking

{{<mermaid>}}
flowchart LR
id1(Track)-->id2(Model)-->id3(Visualise)-->id4(Next steps)
style id1 fill:#f5f5f5,stroke:#6638B8,stroke-width:3px
style id2 fill:#f5f5f5,stroke:#333,stroke-width:1px
style id3 fill:#f5f5f5,stroke:#333,stroke-width:1px
style id4 fill:#f5f5f5,stroke:#333,stroke-width:1px
{{</mermaid >}}

> Get started with consent related events using the JavaScript tracker. The tracking API is made to be simple, understandable and feel familiar with other tracking solutions.

Reviewer: This is regarding other tracking APIs out in the wild. We make sure that we do not diverge on what a Frontend Engineer is used to writting concerning tracking APIs.

> Get started with consent related events using the JavaScript tracker. The tracking API is made to be simple, understandable and feel familiar with other tracking solutions.

Reviewer 2: OK makes sense 👍

> Get started with consent related events using the JavaScript tracker. The tracking API is made to be simple, understandable and feel familiar with other tracking solutions.
> 
> Once set up, you will be able to setup consent preference tracking adjusted to your selected consent management platform (CMP).

Reviewer 3: ```suggestion
Learn to send consent events to your warehouse using the JavaScript tracker and your consent management platform (CMP).
```
